frontend/src/helpers/ServerPath.js

```diff
@@ -0,0 +1,6 @@
+const ServerPath = {
+    api: 'http://localhost:8000/api',
+    thumbnail: 'http://127.0.0.1:8000/storage/'
+};
+
+export default ServerPath;
```

Comment:
For this one. react has feature to utilize environment configuration files. to apply it. please create a `.env` file in the `root` directory of your project and  put this in:
```
REACT_APP_API=http://localhost:8000/api
REACT_APP_THUMBNAIL=http://127.0.0.1:8000/storage/
```
(compile react app again)
with this you can hide this configuration separate from your code.
then from here you can access your configuration  via `process.env.REACT_APP_API` or `process.env.REACT_APP_THUMBNAIL`.
so you wont be need this file anymore. 

note:
also make sure to document all your `env` variables so the next developer can also create a new env file `.env.example` and store this:
```
REACT_APP_API=http://localhost:8000/api
REACT_APP_THUMBNAIL=http://127.0.0.1:8000/storage/
```
also make sure to add `.env` in your gitignore file so it doesnt push this file to the github repository since we want this configuration to by dynamic for its local device.

Reply:
Got it Sir, I'll update this one.